packages/runtime-vapor/src/componentProps.ts

```diff
@@ -370,7 +370,10 @@ function propsDeleteDevTrap(_: any, key: string | symbol) {
 }
 
 export const rawPropsProxyHandlers: ProxyHandler<RawProps> = {
-  get: getAttrFromRawProps,
+  get(target, key) {
+    if (key === ReactiveFlags.RAW) return target
```

Comment:
The changes here are too broad. The actual bug scenario is clear and only exists when passing vapor props to createVNode. I think the changes in #13382 would be safer.